execute/executetest/table.go

```diff
@@ -475,11 +476,58 @@ func (b SortedTables) Swap(i int, j int) {
 	b[i], b[j] = b[j], b[i]
 }
 
-// NormalizeTables ensures that each table is normalized
+// NormalizeTables ensures that each table is normalized and that tables and columns are sorted in
+// alphabetical order for consistent testing
 func NormalizeTables(bs []*Table) {
 	for _, b := range bs {
 		b.Key()
 	}
+	sortByGroupKey(bs)
+}
+
+func sortByGroupKey(tables []*Table) {
+	sort.Sort(SortedTables(tables))
+	for _, table := range tables {
+		sortColumns(table)
+	}
+}
+
+func sortColumns(table *Table) {
+	// index reference to make sure that sort is consistent for metadata and data
+	indexes := createReferenceArray(table)
+
+	columnsCopy := make([]flux.ColMeta, len(table.ColMeta))
+	for i, value := range indexes {
+		columnsCopy[i] = table.ColMeta[value]
```

Comment:
Does `copy` not do this for us?

Reply:
No, the copy doesn't populate the map with the required values. For some reason, the `copy` function doesn't play well with `flux.ColMeta` values.

Reply:
Could you add a comment to that effect? I can see myself seeing that one day and thinking "why is this like this?" and trying to simplify it. Luckily, we'll have a failing test as a result, but the _why_ would be super helpful.

Reply:
Comment added.